### PR TITLE
Update prebuilt rule docs for 8.8

### PR DESCRIPTION
**Pull request by Mikaayenson**
# Summary

Updates the docs per design in https://github.com/elastic/detection-rules/issues/2867#issuecomment-1650370453

### CODE REVIEW

File: docs/detections/prebuilt-rules/rule-desc-index.asciidoc [hunk 1]
```diff
@@ -187,43 +197,63 @@ include::rule-details/disable-windows-firewall-rules-via-netsh.asciidoc[]
...
 include::rule-details/executable-file-creation-with-multiple-extensions.asciidoc[]
 include::rule-details/execution-from-unusual-directory-command-line.asciidoc[]
 include::rule-details/execution-of-com-object-via-xwizard.asciidoc[]
 include::rule-details/execution-of-file-written-or-modified-by-microsoft-office.asciidoc[]
 include::rule-details/execution-of-file-written-or-modified-by-pdf-reader.asciidoc[]
 include::rule-details/execution-of-persistent-suspicious-program.asciidoc[]
-include::rule-details/execution-via-electron-child-process-node.js-module.asciidoc[]
-include::rule-details/execution-via-mssql-xp_cmdshell-stored-procedure.asciidoc[]
+include::rule-details/execution-via-electron-child-process-node-js-module.asciidoc[]
+include::rule-details/execution-via-mssql-xp-cmdshell-stored-procedure.asciidoc[]
```
Comment (Mikaayenson) [inline docs/detections/prebuilt-rules/rule-desc-index.asciidoc:231]: Note: @terrancedejesus some of these references were renamed when building via automation

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 2]
```diff
@@ -5,6 +5,8 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+To see the updates for 8.7 and further releases, please visit the https://www.elastic.co/guide/en/security/8.7/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```
Comment (Mikaayenson) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:8]: @approksiu @terrancedejesus I think we need to remove this page altogether right?

Comment (Mikaayenson) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:8]:
```suggestion
To see the updates for 8.8 and further releases, please visit the https://www.elastic.co/guide/en/security/8.8/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 3]
```diff
@@ -5,6 +5,8 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+To see the updates for 8.8 and further releases, please visit the https://www.elastic.co/guide/en/security/8.8/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```
Comment (Mikaayenson) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:8]:
```suggestion
To see the updates for 8.9 and further releases, please visit the https://www.elastic.co/guide/en/security/8.9/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 4]
```diff
@@ -5,6 +5,8 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+To see the updates for 8.9 and further releases, please visit the https://www.elastic.co/guide/en/security/8.9/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```
Comment (Mikaayenson) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:8]:
```suggestion
To see the updates for 8.8 and further releases, please visit the https://www.elastic.co/guide/en/security/8.8/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```

**Review by approksiu — APPROVED** [review]
LGTM

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc (hunk 2 above)
Comment (approksiu) [reply docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc]: We need to keep this page (and update it to link to the downloadable rule updates per https://user-images.githubusercontent.com/1636709/256029653-0df94b78-fefb-4f86-afea-b4c4f81ea903.png). It is still referenced from security solution on 8.8.

File: docs/detections/detections-index.asciidoc
Comment (approksiu) [inline docs/detections/detections-index.asciidoc:1]: Please do not remove this page.

**Review by approksiu — CHANGES_REQUESTED** [review]

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc
Comment (approksiu) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:1]: keep this page please

**Review by approksiu — CHANGES_REQUESTED** [review]

**Review by approksiu — APPROVED** [review]

**Review by joepeeples — APPROVED** [review]

**Pull request merged**